lib/no_brainer/migration.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module NoBrainer
+  #
+  # Parent class of all migration scripts.
+  # This class is used by the NoBrainer::Migrator class.
+  #
+  class Migration
+    class NoImplementedError < StandardError; end
```

Comment:
Take this out. Use the `NotImplementedError` of ruby

Reply:
I didn't know Ruby had such a class, I will change that!